content/docs/fairing/gcp-ai-platform-notebook.md

```diff
@@ -0,0 +1,121 @@
+   way for you to set up a GKE cluster with Kubeflow
+   installed, or
+*  You can deploy Kubeflow using the [command line][kubeflow-install].
+
+## Set up your AI Platform Notebooks instance
+
+Kubeflow Fairing requires Python 3.6 or later. Currently, only AI Platform
+Notebooks instances created with the PyTorch framework image have Python 3.6
+or later installed. If you do not have an AI Platform Notebooks instance that
+uses the PyTorch framework, follow the guide to [creating a new notebook
+instance][ai-notebook-create] to set up your environment and select the
+PyTorch **Framework**. 
+
+![](/docs/images/fairing/ai-platform-notebook-pytorch.png)
+
+## Run the example notebook
+
+Follow these instructions to set up your environment and run the XGBoost
+quickstart notebook:
+
+1.  Use the AI Platform Notebooks user interface to [open your hosted notebook
+    environment][ai-notebook-open].
+
+1.  Download the files used in this example and install the packages that the
+    XGBoost quickstart notebook depends on.
+
+    1.  In the JupyterLab user interface, click **File** > **New** > **Terminal**
+        in the menu to start a new terminal session in your notebook environment.
+        Use the terminal session to set up your notebook environment to run this
+        example.
+
+    1.  Clone the Kubeflow Fairing repository to download the files used in
+        this example.
+
+        ```bash
+        git clone https://github.com/kubeflow/fairing 
+        cd fairing
+        ```
+
+    1.  Upgrade Kubeflow Fairing from the cloned repository.
+
+        ```bash
+        pip install .
+        ```
+
+    1.  Install the Python dependencies for the XGBoost quickstart notebook.
+
+        ```bash
+        pip install -r examples/prediction/requirements.txt
+        ```
+
+    1.  Authorize Docker to access your [GCP Container Registry][container-registry]. 
+
+        ```bash
+        gcloud auth configure-docker
+        ```
+
+    1.  Update your `kubeconfig` with appropriate credentials and endpoint
+        information for your Kubeflow cluster. To find your
+        cluster's name, run the following command to list the clusters in your
+        project:
+
+        ```bash
+        gcloud container clusters list
+        ```
+
+        Update the following command with your cluster's name and GCP zone, then
+        run the command to update your `kubeconfig` to provide it with credentials
+        to access this Kubeflow cluster.
+    
+        ```bash
+        export CLUSTER_NAME=kubeflow
+        export ZONE=us-central1-a
+        gcloud container clusters get-credentials $CLUSTER_NAME --region $ZONE
+        ```
+
+
+1.  Use the notebook user interface to open the XGBoost quickstart notebook
+    at `[path-to-cloned-fairing-repo]fairing/examples/prediction/xgboost-high-level-apis.ipynb`.
+
+1.  Follow the instructions in the notebook to:
+
+    1.  Train an XGBoost model in a notebook,
```

Comment:
Suggestion: It may be just a personal preference, but I prefer a bulleted list (as you've used at the top of the page) rather than a numbered list.

Reply:
Done